index.html

```diff
@@ -31,16 +31,17 @@ <h1 class="blog-post-title"><a href="{{ post.url }}">{{ post.title }}</a></h1>
         (about {{ minutes }} minutes)
         {% endif %}
       </a>
+	  {% comment %} /* disqus disabled b/c it's flaky (and we're not sure if it's useful) */
```

Comment:
let's keep this file untouched, but leave a comment about the flakiness in the main _config.yml